src/pages/Código bomba.wziwg.js

```diff
@@ -1,7 +1,88 @@
 // API Reference: https://www.wix.com/velo/reference/api-overview/introduction
 // “Hello, World!” Example: https://learn-code.wix.com/en/article/1-hello-world
+import wixStorage from 'wix-storage';
+
+import { utils_config_items, 
+    utils_fmt_only_number,
+    utils_get_elements_values,
+    utils_show_hide_section,
+    SECTION_STATE_LOADING,
+    SECTION_STATE_DATA,
+    SECTION_STATE_NO_DATA,
+} from "public/utils";
+
+import {
+    be_mod_utils_get_bombas_code,
+} from "backend/be_mod_utils";
+
+
+let g_codigo_bomba = [
+    {ui: "#inputCodBomba1", type: "value", format: utils_fmt_only_number, onChange: get_bomba_suggestion},
+    {ui: "#inputCodBomba2", type: "value", format: utils_fmt_only_number, onChange: get_bomba_suggestion},
+    {ui: "#inputCodBomba3", type: "value", format: utils_fmt_only_number, onChange: get_bomba_suggestion},
+    {ui: "#inputCodBomba4", type: "value", format: utils_fmt_only_number, onChange: get_bomba_suggestion},
+    {ui: "#inputCodBomba5", type: "value", format: utils_fmt_only_number, onChange: get_bomba_suggestion},
+];
+
+let g_bombas_map = [
+    {ui: "#textNomePosto", db: "cod_e_posto", type: "text", onClick: onclick_bomba_selected},
+    {ui: "#textEnderecoPosto", db: "endereco_posto", type: "text"},
+    {ui: "#imagePosto", db: "img_posto", type: "src"}
+];
+
+const map_bombas = {"bomba1": "#inputCodBomba1", "bomba2": "#inputCodBomba2", "bomba3": "#inputCodBomba3", "bomba4": "#inputCodBomba4", "bomba5": "#inputCodBomba5"};
+
+
+function set_sections(state) {
+    switch (state){
+        case SECTION_STATE_LOADING:
+            return utils_show_hide_section(["#sectionLoading"], ["#sectionPostos", "#sectionNoData"]);
+        case SECTION_STATE_DATA:
+            return utils_show_hide_section(["#sectionPostos"], ["#sectionLoading", "#sectionNoData"]);
+        case SECTION_STATE_NO_DATA:
+            return utils_show_hide_section(["#sectionNoData"], ["#sectionLoading", "#sectionPostos"]);
+    }
+}
+
+function onclick_bomba_selected(event) {
+    let cod_bomba = event.target.text.split(" - ")[0];
+
+    Array.from(cod_bomba).forEach((char, index) => {
+        $w(map_bombas[`bomba${index + 1}`]).value = char;
+    });
+
+    get_bomba_suggestion(); // to update the suggestions
+    $w("#buttonCodBombaAvancar").enable();
+
+    wixStorage.local.setItem('cod_bomba_value', JSON.stringify(cod_bomba));
```

Comment:
why "local" and not "memory" ?

Reply:
I used 'local' for being the first option that I founded when I was looking for a way to save data. Now that I search about 'memory' maybe it's a better choice, but I'm cleaning local storage every time that user goes to home page, so maybe will be a performance topic (added check this as a TODO option, just because I'm  cleaning local storage)